homeassistant/components/wallbox/const.py

```diff
@@ -2,6 +2,7 @@
 from enum import StrEnum
 
 DOMAIN = "wallbox"
+UPDATE_INTERVAL = 30
```

Comment:
Please keep constants that are not shared between different modules in the module where the constant is used.

Reply:
Will do the next time, in this case `UPDATE_INTERVAL` is used in `__init__.py` (init. the library, which isn't in the diff) and in  `coordinator.py`

Reply:
And `CHARGER_STATUS`?

Reply:
Yea this should be kept in the module. Should I move this back?

Reply:
Please move it to the coordinator module.